nrf_security/src/mbedcrypto_glue/CMakeLists.txt

```diff
@@ -3,55 +3,66 @@
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
-if(CONFIG_NRF_CRYPTO_GLUE_LIBRARY)
-  zephyr_include_directories(
-      ${NRF_MBEDTLS_ROOT}/include/mbedcrypto_glue/mbedtls
-  )
 
+if(CONFIG_NRF_CRYPTO_GLUE_LIBRARY)
   zephyr_library_named(mbedcrypto_glue)
-  zephyr_library_sources(
-      aes_alt.c
-      ccm_alt.c
-      dhm_alt.c
-      ecdh_alt.c
-      ecdsa_alt.c
-      platform_alt.c
-      rsa_alt.c
-  )
 
+  #
+  # Add files to glue if they are glued
+  #
+  zephyr_library_sources_ifdef(CONFIG_GLUE_MBEDTLS_AES_C    aes_alt.c)
+  zephyr_library_sources_ifdef(CONFIG_GLUE_MBEDTLS_CCM_C    ccm_alt.c)
+  zephyr_library_sources_ifdef(CONFIG_GLUE_MBEDTLS_DHM_C    dhm_alt.c)
+
+  #
+  # Add include directories shared by all glue libraries
+  #
   zephyr_library_include_directories(
-      ${NRF_MBEDTLS_ROOT}/include/mbedcrypto_glue
+      "${NRF_MBEDTLS_ROOT}/include/mbedcrypto_glue"
+      "${common_includes}"
```

Comment:
Isn't this redundant with https://github.com/NordicPlayground/nrfxlib/pull/82/files#diff-b34bd274b90e72abf1c2550f34011a26R19 ?

Reply:
Fixed